src/main/java/com/naveenautomation/Pages/ShoppingCartPage.java

```diff
@@ -12,6 +13,19 @@ public ShoppingCartPage(WebDriver wd, boolean waitForPageToLoad) {
 		super(wd, waitForPageToLoad);
 	}
 
+	private static final By estimateShippingAndTaxesLink = By.cssSelector("div.panel-group>div:nth-of-type(2) a");
+	private static final By countryDropDown = By.id("input-country");
+	private static final By regionDropDown = By.id("input-zone");
+	private static final By postalCodeInputBox = By.cssSelector("input#input-postcode");
+	private static final By getQuotesBtn = By.cssSelector("button#button-quote");
+	private static final By shippingRateRadioBtn = By.cssSelector("input[type='radio']");
+	private static final By applyShippingBtn = By.cssSelector("input#button-shipping");
+	private static final By alertBanner = By.cssSelector("div.alert");
+
+	public void clickestimateShippingAndTaxesLink() {
```

Comment:
is this link opening a page, or dialog, if yes then it should not be void

Reply:
Further to our discussion, I have changed the name to expandEstimateShippingTaxes